app/commands/donations/paypal/subscription/handle_recurring_payment.rb

```diff
@@ -0,0 +1,23 @@
+# Handle a Paypal IPN event with 'txn_type' = 'recurring_payment'
+class Donations::Paypal::Subscription::HandleRecurringPayment
+  include Mandate
+
+  initialize_with :payload
+
+  def call
+    user = Donations::Paypal::Customer::FindOrUpdate.(payer_id, payer_email)
+    return unless user
+
+    subscription = Donations::Paypal::Subscription::Create.(user, subscription_external_id, amount)
+    return unless subscription
```

Comment:
In principle, we should already have a subscription due to the `recurring_payment_profile_created` happening before, but to guard against us missing that event _and_ to help with getting the existing customers in the DB, this will also automatically create a subscription if it is not there